src/main/java/hudson/plugins/junitattachments/AttachmentTestAction.java

```diff
@@ -63,7 +63,7 @@ public static boolean isImageFile(String filename) {
 	}
 
 	public static String getUrl(String filename) throws UnsupportedEncodingException {
-		return "attachments/" + URLEncoder.encode(filename, "UTF-8");
+		return "attachments/" + URLEncoder.encode(filename, "UTF-8").replace("+", "%20");
```

Comment:
Does `URLEncoder.enocde` not cover this?

```
System.out.println(URLEncoder.encode("hello+jenkins", "UTF-8"));

= > hello%2Bjenkins
```

Reply:
Without this changes, I had a "+" sign, and all links with spaces were broken, so these changes fix my problem.

Reply:
I suspect you meant to use [this](http://javadoc.jenkins.io/hudson/Util.html#rawEncode-java.lang.String-) more generally, and of course it should be reproduced in a test, preferably in a separate PR making only no changes and no others.

Reply:
I tried, bit it did not work because other chars escaped different and I only need to replace  a space from %20 space to +